proto/raft.proto

```diff
@@ -294,6 +294,25 @@ enum FinalizeOperation {
   COMMIT = 2;
 }
 
+// A client session that is used to support idempotency on the raft state
+// machine.
+message Session {
+  // An unique identifier of the client session.
+  // Example: 018fcb2f-cfab-718b-974e-64907df56881
+  bytes id = 1;
+  // Each BatchCmdRequest has its unique <id, index>. The index is
+  // monotonically increasing.
+  // Example: 1717100891000000000
+  uint64 index = 2;
+  // A timestamp when the session will be cleaned up; and when the state machine
+  // can safely clean up the entry.
+  int64 expired_at = 3;
```

Comment:
s/expired_at/expiry/g

And can we specify units in the comment?

Reply:
I renamed to expiry_usec.